src/pages/batchImports/[resourceId]/index.js

```diff
@@ -213,6 +213,7 @@ const BatchImports = () => {
           },
           width: 500,
           height: 450,
+          closable: false,
```

Comment:
did you make sure you don't need draggable:flase?

Reply:
it can be draggable